### PR TITLE
Mention `behat -dl` as a useful way of seeing defined steps

### PR DESCRIPTION
**Pull request by danielbachhuber**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,8 +106,16 @@ Functional tests typically follow this pattern:
 * **When** a user performs a specific action,
 * **Then** the end result should be X (and Y and Z).
 
-This command generates all of the files needed for you to write Behat
-tests for your own command. Specifically:
+View all defined Behat steps available for use with `behat -dl`:
+
+```
+Given /^an empty directory$/
+Given /^an empty cache/
```
Comment (schlessera) [inline README.md]: Maybe `$` missing at the end?

File: src/ScaffoldPackageCommand.php [hunk 2]
```diff
@@ -370,9 +370,9 @@ public function package_readme( $args, $assoc_args ) {
 	 * Generate files needed for writing Behat tests for your command.
 	 *
 	 * WP-CLI makes use of a Behat-based testing framework, which you should use
-	 * too. Functional tests are an integral ingredient of highly-quality,
-	 * maintainable commands. Behat is a great choice as a testing framework
-	 * because:
+	 * too. This command generates all of the files you need. Functional tests
+	 * are an integral ingredient of highly-quality, maintainable commands.
```
Comment (schlessera) [inline src/ScaffoldPackageCommand.php]: Again "high-quality"

File: src/ScaffoldPackageCommand.php [hunk 3]
```diff
@@ -401,8 +401,16 @@ public function package_readme( $args, $assoc_args ) {
 	 * * **When** a user performs a specific action,
 	 * * **Then** the end result should be X (and Y and Z).
 	 *
-	 * This command generates all of the files needed for you to write Behat
-	 * tests for your own command. Specifically:
+	 * View all defined Behat steps available for use with `behat -dl`:
+	 *
+	 * ```
+	 * Given /^an empty directory$/
+	 * Given /^an empty cache/
```
Comment (schlessera) [inline src/ScaffoldPackageCommand.php]: Another potential `$` missing at the end.

File: README.md [hunk 4]
```diff
@@ -75,9 +75,9 @@ wp scaffold package-tests <dir> [--ci=<provider>] [--force]
 ~~~
 
 WP-CLI makes use of a Behat-based testing framework, which you should use
-too. Functional tests are an integral ingredient of highly-quality,
-maintainable commands. Behat is a great choice as a testing framework
-because:
+too. This command generates all of the files you need. Functional tests
+are an integral ingredient of highly-quality, maintainable commands.
```
Comment (danielbachhuber) [inline README.md]: Fixed 4929dd1

File: src/ScaffoldPackageCommand.php (hunk 2 above)
Comment (danielbachhuber) [inline src/ScaffoldPackageCommand.php]: Fixed 4929dd1

File: README.md (hunk 1 above)
Comment (danielbachhuber) [inline README.md]: This was generated directly from `behat -dl`

**Comment by danielbachhuber** [comment]
@schlessera Fixed

**Pull request merged**